### PR TITLE
Nerf border changing to pointer

### PR DESCRIPTION
**Pull request by Incompleteusern**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java [hunk 1]
```diff
@@ -268,8 +268,8 @@ private void renderPois(
             compassOffsetX /= distance;
             compassOffsetZ /= distance;
 
-            float scaledWidth = width - 4 * compassSize;
-            float scaledHeight = height - 4 * compassSize;
+            float scaledWidth = width - 2 * compassSize;
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java:271]: Are we maybe missing a parenthesis? Or am I tired..

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java:271]: What parenthesis?

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java:271]: Only the numbers have changed from 4 to 2

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java:271]: (width -2) * compassSize?

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/features/user/overlays/map/MinimapFeature.java:271]: No, that is not desired as we are subtracting from width

**Pull request merged**